spec/models/author_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Author, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
add model level validations, check length, uniqueness validations and assert the errors n success cases

Reply:
Added test cases please check latest commit.